### DISCUSSION TITLE
How can I use "disabled" property?

### DISCUSSION THREAD

**Discussion by Kikamilka** [sibiraj-s/ngx-editor#439] [Q&A] [answered]
I see in the documentation that for readonly mode I can use [disabled] property:
![image](https://user-images.githubusercontent.com/7958264/197569053-f38c3659-93fd-4cec-8dfe-c3df9f14fc2a.png)
But at the same time, I can't add this property in my application - I see error 'Unknown property'. And when I try to find this component in your repository, I also found that "disabled" doesn't exist as Input() property in [NgxEditorComponent](https://github.com/sibiraj-s/ngx-editor/blob/master/projects/ngx-editor/src/lib/editor.component.ts) 
Please, could you help me with it and describe how exactly I can use this property and where?

**Comment by sibiraj-s** [comment]
The editor uses ControlValueAccessor to make it work like the html input. See here https://github.com/sibiraj-s/ngx-editor/blob/master/projects/ngx-editor/src/lib/editor.component.ts#L63.

Can you provide a reproducible example in stackblitz to check this?

**Reply by Kikamilka** [reply]
https://stackblitz.com/edit/angular-cdztrc?file=src/app/editor.component.ts
I tried to create something that similar for using editor in our application.

As I understood after writing the example, the problem with "disabled" property can be in missed [ngModel] in my example, right? 
But I don't fully understand for which cases we need this ngModel and what exactly should be there as initial state?

**Reply by sibiraj-s** [reply, marked as answer]
Sorry for the delay in reply. The editor input component tries to behave like a form input component. To bind a value to the component you will need ngModel. The iniital state can be a string(HTML) or a JSON output of the editor.


See [outputFormat](https://sibiraj-s.github.io/ngx-editor/en/configuration/#component-props) to configure the output formats for the editor

[a comment by an automated account is not shown]
